js/dialog.js

```diff
@@ -0,0 +1,282 @@
+/**
```

Comment:
This file seems to be an extended version of src/dialog.js. Please update accordingly.